pre_commit/languages/docker.py

```diff
@@ -8,13 +10,43 @@
 from pre_commit.languages import helpers
 from pre_commit.prefix import Prefix
 from pre_commit.util import clean_path_on_failure
+from pre_commit.util import cmd_output_b
 
 ENVIRONMENT_DIR = 'docker'
 PRE_COMMIT_LABEL = 'PRE_COMMIT'
 get_default_version = helpers.basic_get_default_version
 healthy = helpers.basic_healthy
 
 
+def _is_in_docker() -> bool:
+    try:
+        with open('/proc/1/cgroup', 'rb') as f:
+            return b'docker' in f.read()
+    except FileNotFoundError:
+        return False
+
+
+def _get_docker_path(path: str) -> str:
+    if not _is_in_docker():
+        return path
+    hostname = socket.gethostname()
+
+    _, out, _ = cmd_output_b(
+        ' '.join(['docker', 'inspect', hostname]),
+    )
```

Comment:
this should be `cmd_output_b('docker', 'inspect', 'hostname')`

Reply:
done. Function signature is a bit misleading then =(

Reply:
what is misleading about it?  it collects strings into `cmd` therefore `*cmd`

Reply:
Because "cmd" is usually "a cmd", single command line. If it accepts many things, "args" or something similar would have been clearer IMO. But nevermind.